### PR TITLE
RSDK-7258: change status to message type in board api

### PR DESCRIPTION
**Pull request by martha-johnston**

### CODE REVIEW

**Review by susmitaSanyal — APPROVED** [review]

File: proto/viam/component/board/v1/board.proto
Comment (martha-johnston) [inline proto/viam/component/board/v1/board.proto:116]: it's part of the scope to remove Analog and Digital statuses, however it is breaking app. I think it's fixable, but it requires changes to the TS sdk and the TS sdk cannot be tested until these api changes are merged. I'd like to start with this non-breaking merge and then update typescript and then come back and make the final api change to remove the analog and digital statuses. thoughts @susmitaSanyal

**Comment by martha-johnston** [comment]
@Simberific do you have to merge this since it's breaking? If it's someone else let me know!

File: proto/viam/component/board/v1/board.proto
Comment (susmitaSanyal) [reply proto/viam/component/board/v1/board.proto:116]: sounds good!

**Review by Simberific — APPROVED** [review]
LGTM

**Pull request merged**
